NYPLAudiobookToolkit/UI/ScrubberView.swift

```diff
@@ -163,6 +163,7 @@ final class ScrubberView: UIView {
         self.topLabel.text = self.state.progress.timeLeftInBookText
         self.middleLabel.text = self.state.middleText
         self.setNeedsUpdateConstraints()
+        self.layoutIfNeeded()
```

Comment:
why do we need `layoutIfNeeded()`

Reply:
`layoutIfNeed()` will force a layout pass immediately instead of waiting for the next run loop. Waiting for the next run loop means a user sees a "glitch" or jump to the actual place we want the views to do. This is always why you said this paired with `setNeedsLayout()` in Apple conformance, but in this case pairing it with `setNeedsUpdateConstraints()` is equivalent.